modules/migration-mtc-release-notes-1-8.adoc

```diff
@@ -46,6 +46,33 @@ In this release, on migrating an application including a `BuildConfig` from a so
 In this release, after enabling `Require SSL verification` and adding content to the CA bundle file for an MCG NooBaa bucket in MigStorage, the connection fails as expected. However, when reverting these changes by removing the CA bundle content and clearing `Require SSL verification`, the connection still fails. The issue is only resolved by deleting and re-adding the repository. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2240052[*BZ#2240052*])
 
 
+.Backup phase fails after setting custom CA replication repository
+
+In {mtc-full} ({mtc-short}), after editing the replication repository, and adding a custom CA, and the repo is successfully connected, when triggering a migration, there is a failure returned on the backup phase.
+
+This issue has been resolved in in {mtc-short} 1.8.2.
+
+
+.CVE-2023-26136: tough-cookie package before 4.1.3 are vulnerable to Prototype Pollution
+
+In {mtc-short}, versions of the package tough-cookie before 4.1.3, used in {mtc-short}, are vulnerable to Prototype Pollution because of the improper handling of Cookies when using CookieJar in `rejectPublicSuffixes=false` mode.
+
+It is advised to update to {mtc-short} 1.8.2 or later, which resolve this issue.
+
+For more details, see link:https://access.redhat.com/security/cve/cve-2023-26136[(CVE-2023-26136)]
+
+
+.CVE-2022-25883 openshift-migration-ui-container: nodejs-semver: Regular expression denial of service
+
+In previous releases of ({mtc-short}), versions of the package semver before 7.5.2, used in {mtc-short}, are vulnerable to Regular Expression Denial of Service (ReDoS) from the function new Range, when untrusted user data is provided as a range.
+
+It is advised to update to {mtc-short} 1.8.2 or later, which resolve this issue.
```

Comment:
```suggestion
In previous releases of ({mtc-short}), versions prior to 7.5.2 of the `semver` package used in {mtc-short} were vulnerable to Regular Expression Denial of Service (ReDoS) attacks when untrusted user data was provided as a range in the new `Range` function.

To resolve the issue, update to {mtc-short} 1.8.2 or later.
```

I have the same comments and suggested edits as in the previous nearly identical item.